cmd/node/node_test.go

```diff
@@ -914,6 +914,90 @@ func TestConfig_GenesisAccounts(t *testing.T) {
 	})
 }
 
+type nonFatalLogger struct {
+	log.Log
+	record []string
+}
+
+func TestGenesisConfig(t *testing.T) {
+	t.Run("config is written to a file", func(t *testing.T) {
+		app := New()
+		app.Config = &config.Config{
+			BaseConfig: config.BaseConfig{
+				DataDirParent: t.TempDir(),
+			},
+			Genesis: &config.GenesisConfig{
+				ExtraData:   "test",
+				GenesisTime: time.Now().Format(time.RFC3339),
+			},
+		}
+		require.ErrorContains(t, app.Initialize(), "tortoise")
```

Comment:
maybe you can start from DefaultTestConfig() and modify the genesis config and data dir so you don't hit the tortoise param error. see `TestInitialize_BadTortoiseParams`

Reply:
thanks, slipped from my mind